docs/HOWTOS.md

```diff
@@ -81,4 +82,51 @@ In the output of amblist, locate the user data section and find the data that th
           DATE         USER DATA
 CSECT:    EPSCMORT
           06/19/2023   MortgageApplication/a21b6ab0
-```
\ No newline at end of file
+```
+
+## Reduce necessary **application-conf** configurations within the application repository
+
+**The 3 properties for loading configuration files to configure the build framework**
+
+zAppBuild provides 3 properties `buildPropFiles`, `applicationDefaultPropFiles` and the `applicationPropFiles` that each reference a list of properties files, which contain important configuration parameters to configure the build process. The referenced properties files are loaded in the below order of precedence: 
+
+* `buildPropFiles` managed in [build-conf/build.properties](../build-conf/build.properties) references properties files in the [build-conf](../build-conf/) directory for core zAppBuild settings for the language scripts such as system datasets, naming conventions of build datasets, dataset characteristics and various core properties of the build framework as well, like the reporting features.
+* The property `applicationDefaultPropFiles` is managed in [build-conf/build.properties](../build-conf/build.properties) as well and allows the user to define default application (and also language script specific related) properties, that are centrally managed. 
+* `applicationPropFiles` is referencing properties files providing application specific settings. This property is managed in the applications' `application-conf/application.properties` file which is by default located in the applications' [application-conf](../samples/application-conf/) folder. These settings define compiler and link options, deploy types, script mappings and search path configurations for the DBB dependency and impact analysis. A sample is provided in [application-conf](../samples/application-conf/). The default location for zAppBuild to locate the `application.properties` file within the `application-conf` directory is defined via the `applicationConfDir` setting that is managed in [build-conf/build.properties](../build-conf/build.properties). The default mandates am `application-conf` folder including the `application.properties` file to be present in the applications' git repository.
```

Comment:
Same comment - Can we say these are "application-level" properties and that they override the default properties defined through `applicationDefaultPropFiles`?